### PR TITLE
Cisco umbrella fixes

### PR DESCRIPTION
**Pull request by asafshen**
- Cisco Umbrella Integration Fixes
- Cisco Umbrella Playbook Test
- VerifyContext Script utility

### CODE REVIEW

**Comment by Spivaka** [comment]
you have another pr for umbrella fixes, this one is still required??

File: Scripts/VerifyContext.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  var isArray = contextObject instanceof Array;
+
+  if(isArray) {
+      contextObject.forEach(verifyFields);
+  } else {
+      verifyFields(contextObject);
+  }
+type: javascript
+tags:
+- Utilities
```
Comment (slavikm) [inline Scripts/VerifyContext.yml]: Wrong tag?

File: Scripts/VerifyContext.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+enabled: true
+args:
+- name: path
+  required: true
+  default: true
+  description: Context path (e.g. 'Incidents')
+- name: fields
+  description: Comma separated values to verify that are in context(e.g. 'one,to,three')
```
Comment (slavikm) [inline Scripts/VerifyContext.yml]: to -> two

File: Scripts/VerifyContext.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+  var verifyFields = function(obj) {
+      fields.forEach(function(field) {
+          if(obj[field] === undefined) {
+              throw 'field ' + field + ' is missing from ' + JSON.stringify(obj);
+          }
+      });
+  }
+
+  var contextObject = verifyContextField(path);
```
Comment (slavikm) [inline Scripts/VerifyContext.yml]: What is `verifyContextField`? Not sure I've seen it in common

File: Integrations/integration-Cisco-umbrella.yml [hunk 4]
```diff
@@ -1,16 +1,16 @@
...
...U9fSTBf54mjQCbtPz798xh5MmvuXmg+kfE4Kc/5zXvp/lBee/wM+LovXtSLrc2huYNPE3zRBnOZ5uh5oqBm8Wy6YLtr8uL+e3edfXrvt5jxaz0Gbp+WeuiHk9bnClNH4SiYp74tuVZoLL7jqvKtv+n+HF/xvd2s0z9NzaGzu2qzncPof0DHB6QMAAAAOlH4fXr8vz3Tzel23Ce0Cj7j8Rk4ixNwo1j3vLQ8/b+T+ezh0weU+LIhxC1rxIuUVKT8wk5QzjNt9g85Jbcr0F8r0913O+8r09zUwH5pj5yhCzsXLlKNpOK/HuUinLH2W10/HUha6/zblhKZ0fv389dWN5V9ZfcJrX6K1RzZFc0c9v6XVaF+FU64HSH89IJ8+K9xiBwAAAOwNZtoAAADAPjNbl8FbM92Yjl75sLe5Wz57rVaRThG1krRzuqTS7aOLLHocBFoVfO3Dv//20d2+6aH5zHVrIHpeY/cj/bnU6uG09QVwBtD/JZrMvP60yEmOCgAA/wzd3N5Z1871Z9I8/ehO51/0VnfL5x5pu85OfQchHID8PZ2rvqOTbxp36G9/798OL3/cg93s6lrfwbfv9hvDVt4HAAAA4KAlAdN1m5iL1sWZqb/Nyzn68aG7PvQH7Gf+8ckhvaE5Q5/v4cyVzpVJ5ylqnjeZtFfCw2SyvlG9/z+U5knn+W6pTnO5U/J21vEAAACAA8lrkbp+bq+PXX7X7900fR6n0/33vuCy4x96/8/4uh6xjnfwLX7mz75v8Yp/8TTT5Xhy49aiXYjkdwAAAMAdgZU0AAAAYJ+5PBGvRXyTO/sE67Tjrjv/yc97uxy54Jm32oGMB0PxdTDOFBJLZ8b6vWjT90qR0EbIlcJFKh+kCnV67nNgl+cntjFtmTvtmdv15opljiBFG7qvUbT698kc3rvc0Uk2Llw/79LFD5hvf8fVRpt05T8TRctZ5JIWhZWxdvDVCu96nmWS3m84AQDuMFq5YsBhAABge0yRezf53AXWp1yv+ylysyY5esMr3mmvfMRLN8vCbMZGgp+I006vKafQrk21jTmaLrSLUO4k1OUh6a0cvWBMDu0cFdlLt2t9Hl35NqKUOcqmjSLWUqXPB0UZYqikPn7B8fPudfi/lv/qP1051uOvPZx0s6M+phPig3ZgtjPFLNgQBwAAAJzpYv+Pr9s5vBYusLomlxI743Lf
```
Comment (slavikm) [inline Integrations/integration-Cisco-umbrella.yml]: ?

File: Integrations/integration-Cisco-umbrella.yml [hunk 5]
```diff
@@ -567,7 +570,9 @@ script:
     outputs:
     - contextPath: Domain.Name
       description: Domain name
-    - contextPath: Domain.Score
+    - contextPath: Domain.Related.Name
+      description: Related domain name
+    - contextPath: Domain.Related.Score
```
Comment (slavikm) [inline Integrations/integration-Cisco-umbrella.yml]: Aren't we supposed to add to malicious as well in certain cases?

File: Integrations/integration-Cisco-umbrella.yml (hunk 4 above)
Comment (asafshen) [inline Integrations/integration-Cisco-umbrella.yml]: Ido noted on some slack channel that all integrations from the same vendor should be with the same image (Cisco Umbrella & Cisco Meraki for example)

File: Integrations/integration-Cisco-umbrella.yml (hunk 5 above)
Comment (asafshen) [inline Integrations/integration-Cisco-umbrella.yml]: We do: `umbrela-domain-search`, `umbrela-domain-categorization`, `umbrella-ip-malicious-domains`

Do you think we need to do this somewhere else as well?

**Comment by meirwah** [comment]
@asafshen can you update from master?

**Comment by meirwah** [comment]
@Spivaka can you review fixes?

File: Integrations/integration-Cisco-umbrella.yml (hunk 5 above)
Comment (slavikm) [inline Integrations/integration-Cisco-umbrella.yml]: If you have malicious you need to add important - please add it after @aviadl fixes to important are finished and merged

**Comment by asafshen** [comment]
@Spivaka There is no other umbrella-fix pr, waiting for your review

**Pull request merged**